### PR TITLE
Missing providers list on module gives error

### PR DESCRIPTION
**Pull request by kamilkisiela**
@dotansimha I need a repro

### CODE REVIEW

**Comment by dotansimha** [comment]
@kamilkisiela 
```
const module = createModule({
  // no providers field at all
});
```

File: packages/graphql-modules/tests/providers.spec.ts [hunk 1]
```diff
@@ -11,6 +11,66 @@ import {
...
+        lorem(
+          _parent: {},
+          _args: {},
+          { injector }: GraphQLModules.ModuleContext
+        ) {
+          return injector.get(Data).lorem();
+        },
+      },
+    },
+  });
```
Comment (kamilkisiela) [inline packages/graphql-modules/tests/providers.spec.ts:42]: No providers

Comment (kamilkisiela) [reply packages/graphql-modules/tests/providers.spec.ts:42]: @dotansimha

Comment (dotansimha) [reply packages/graphql-modules/tests/providers.spec.ts:42]: weird. it failed for me :X maybe it's something with alpha-2 and it was already fixed?

Comment (kamilkisiela) [reply packages/graphql-modules/tests/providers.spec.ts:42]: I will release a patch tomorrow

**Pull request merged**